script/011.1_SetUpgradableMSCAFactoryPlugins.s.sol

```diff
@@ -39,14 +38,14 @@ contract SetUpgradableMSCAFactoryPlugins is Script {
         uint256 key = vm.envUint("MSCA_FACTORY_OWNER_PRIVATE_KEY");
 
         // Initialize setPlugins exec call data
-        uint256 numPlugins = 4;
+        uint256 numPlugins = 3;
         address[] memory plugins = new address[](numPlugins);
         bool[] memory pluginPermissions = new bool[](numPlugins);
 
-        plugins[0] = SINGLE_OWNER_PLUGIN_ADDRESS;
-        plugins[1] = COLD_STORAGE_ADDRESS_BOOK_PLUGIN_ADDRESS;
-        plugins[2] = WEIGHTED_MULTISIG_PLUGIN_ADDRESS;
-        plugins[3] = DEFAULT_TOKEN_CALLBACK_PLUGIN_ADDRESS;
+        plugins[0] = COLD_STORAGE_ADDRESS_BOOK_PLUGIN_ADDRESS;
+        plugins[1] = WEIGHTED_MULTISIG_PLUGIN_ADDRESS;
+        plugins[2] = DEFAULT_TOKEN_CALLBACK_PLUGIN_ADDRESS;
+        // plugins[3] = SINGLE_OWNER_PLUGIN_ADDRESS;
```

Comment:
nitpick: removing this commented out line `// plugins[3]`? we can add it if needed in the future